tests/Console/E2ETest.php

```diff
@@ -90,7 +90,7 @@ public function test_it_runs_configure_command_if_no_configuration()
      * @dataProvider e2eTestSuiteDataProvider
      * @group e2e
      */
-    public function test_it_runs_an_e2e_test_with_success(string $e2eTestName, string $fullPath)
+    public function test_it_runs_an_e2e_test_with_success(string $fullPath)
```

Comment:
`: \Generator`

Reply:
This is a test function, not a dataprovider, so no return type needed (except maybe void in the future)

Reply:
sorry, my mistake